Enterprise-CMCS/eAPD
# Nz/4183 FFP Bug

Resolves #4183

### Description
Some numbers (specifically 28, 29, 56, 57) change the input values (# shown in the field, not necessarily what is saved in the database).

### Significant changes or possible side effects
Changed where the value was being set (in the controller) and then in the component, use `.toFixed(0)` to keep it whole numbers.

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| The FFP table in Budget and FFP in activites | User types in any number (specifically 28, 29, 56, and 57) | The number stays whole. | jest |

### Steps to manually verify this change

1. Go to Budget and FFP page
2. Type in percentage cells 28, 29, 56, and 57
3. Confirm numbers are consistent and correct based on user input.

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
